ajax/libs/proteic/0.0.27/lighter.css

```diff
@@ -0,0 +1 @@
+@import url("https://fonts.googleapis.com/css?family=Montserrat");.proteic .axis text{font:"Varela Round",sans-serif}.proteic .x.axis path{stroke:#f7f7f7;stroke-width:2px;opacity:1}.proteic .x.axis line{stroke-dasharray:1,5}.proteic .x.axis text{fill:#fafafb}.proteic .x.axis line{stroke:#f7f7f7}.proteic .x.axis .tick:nth-of-type(even) line{stroke:#fafafb}.proteic .x.axis .tick:nth-of-type(even) text{fill:#f7f8fa}.proteic .x.axis.linear .tick:nth-of-type(odd) text{fill:#fafafb}.proteic .x.axis.linear .tick:nth-of-type(odd) line{stroke:#f7f7f7}.proteic .xaxis-title{fill:#f7f8fa}.proteic .y.axis path{display:none}.proteic .y.axis .tick:nth-child(even) line{stroke:#fafafb}.proteic .y.axis .tick:nth-child(even) text{fill:#f7f8fa}.proteic .y.axis .tick:nth-child(odd) text{fill:#fafafb}.proteic .y.axis .tick:nth-child(odd) line{stroke:#f7f7f7}.proteic .y.axis .tick line{stroke-dasharray:1,5}.yaxis-title{fill:#f7f8fa}.text-indicator{white-space:nowrap;overflow:hidden}.text-indicator .value{font:2em "Varela Round",sans-serif,sans-serif}.text-indicator .label{font:1em "Varela Round",sans-serif,sans-serif;text-overflow:ellipsis}.proteic .legend{font:"Varela Round",sans-serif}.proteic .legend text{fill:#f7f8fa}:root{--annotation-color:#f7f7f7}.annotation path{stroke:var(--annotation-color);fill:none}.annotation text,.annotation .annotation-connector .connector-dot,.annotation path.connector-arrow,.annotation.callout.rect path.subject{fill:var(--annotation-color);font:"Varela Round",sans-serif}.annotation-note-title{font-weight:bold}.annotation-note-bg{fill:rgba(255,255,255,0)}.annotation.callout.rect path.subject{fill-opacity:.1}.annotation.badge path.subject-pointer,.annotation.badge path.subject{fill:var(--annotation-color);stroke-width:3px;stroke-linecap:round}.annotation.badge path.subject-ring{fill:white;stroke-width:3px}.annotation.badge .badge-text{fill:white;font-size:.7em}.editable .annotation-subject,.editable .annotation-note{cursor:move}circle.handle{stroke-dasharray:5;stroke:var(--annotation-color);fill:rgba(255,255,255,0.2);cursor:move;stroke-opacity:.4}.proteic .alert{fill:#FF7900;stroke:#FF4F00;stroke-width:2}.proteic .box text{font:"Varela Round",sans-serif;font-weight:bold;fill:#1a2127;text-shadow:-1px -1px 0 #f7f8fa,1px -1px 0 #f7f8fa,-1px 1px 0 #f7f8fa,1px 1px 0 #f7f8fa}
```

Comment:
I think we don't need to add this file, what do you think?
Because this file just for one of the example https://github.com/proteus-h2020/proteic/blob/development/examples/barchart.html#L15
Also there is no tutorial show this is the necessary file.

Thanks.